plugins/music_service/spotify/index.js

```diff
@@ -296,7 +296,7 @@ ControllerSpop.prototype.listPlaylists=function()
 	var commandDefer=self.sendSpopCommand('ls',[]);
 	commandDefer.then(function(results){
 			var resJson=JSON.parse(results);
-            //self.logger.info(JSON.stringify(resJson));
+                        self.logger.info(JSON.stringify(resJson));
```

Comment:
Yes, I'm going to do that for the final 2.0 release version.  Plus add better error checking!